packages/main/types/requests.ts

```diff
@@ -99,7 +107,17 @@ export interface StartChatParams extends BaseParams {
  * @public
```

Comment:
If you resolve the question of what happens and what's supposed to happen if you supply both, we should document it here (it throws an error, it prefers one or the other, it combines them in some weird way, or whatever the answer is)

Reply:
Consulted the group and an error when both are defined is preferred.